hbp_knowledge/anatabine/paris2013.bel

```diff
@@ -146,115 +160,149 @@ anatabine (Mann–Whitney U=0, Z=-2.309, P=0.021) and a significant
 inhibition of p65 NFkB phosphorylation was observed with
 600 mg/ml of anatabine (Mann–Whitney U=1.0, Z=-2.021,
 P=0.043) and with 800 mg/ml of anatabine (Mann–Whitney U=0,Z=-2.309, P=0.021)"
+SET Confidence = "High"
 SET CellLine = "SH-SY5Y"
 # Dosage: 600 mg/ml and 800 mg/ml anatabine
 a(CHEBI:Anatabine) -| p(HGNC:STAT3,pmod(Ph))
 a(CHEBI:Anatabine) -| p(HGNC:RELA,pmod(Ph))
 UNSET CellLine
+UNSET Confidence
 
 SET Evidence = "We next tested the impact of anatabine on STAT3 and p65
 NFkB phosphorylation induced by a 24 h treatment with LPS on
 human microglial cells, a cell type known to express alpha7-nicotinic
 acetylcholine receptor subtype (Suzuki et al., 2006)"
+SET Confidence = "Medium"
+SET Species = "9606"
 SET Cell = "microglial cell"
 a(MESH:Microglia) -> a(MESH:"alpha7 Nicotinic Acetylcholine Receptor")
 UNSET Cell
+UNSET Species
+UNSET Confidence
 
+#not sure of information provide: was LPS given with anatabine?
```

Comment:
LPS is lipopolysaccharide
I'm not sure if I understand your concerns, but the evidence only says that treatment with LPS increases phosphorylation of those two proteins. So I don't think anatabine is involved here